src/components/List/drop-down-list.scss

```diff
@@ -0,0 +1,92 @@
+@mixin icon {
+    position: absolute;
+    top: 33%;
+    left: 70%;
+    color: #6a687e;
+    cursor: pointer;
+    height: 20px;
+    width: 20px;
+  }
+  @mixin dropitem {
+    color: black;
+    padding: 12px 16px;
+    cursor: pointer;
+    font-family: Verdana;
+    font-size: 20px;
+    text-align: left;
+  }
+  .container {
+   width: 100%;
+   height:800px;
+  }
+
+  .inputContainer {
+    padding: 5px;
+    align-items: center;
+  
+    input {
+      height: 7%;
+      width: 40%;
+      position: absolute;
+      top: 30%;
+      left: 30%;
+      border-radius: 5px;
+      border-style: solid;
+      border-color: #eaecf2;
+      border-width: 3.5px;
+      font-family: Verdana;
+      font-size: 20px;
+      padding-left: 20px;
+      outline: none;
+  
+      &::placeholder {
+        color: black;
+      }
+    }
+    &.placecolor input::placeholder {
+      color: #696969;
+    }
+    .fa {
+      @include icon;
+    }
+  
+    .fa1 {
+      transform: rotate(180deg);
+      @include icon;
+    }
+  }
+  .list {
+    width: 41%;
+    position: absolute;
+    top: 38%;
+    left: 30.3%;
+    margin-top: 5px;
+    border-radius: 5px;
+    background-color: white;
+    border: 3px solid #eaecf2;
+    max-height: 170px;
+    overflow: auto;
+  
+    &::-webkit-scrollbar {
```

Comment:
why do you need the webkit here ?

Reply:
I saw in the task video that scrollbar thumb disappear after scrolling , so I used it to add this action